frontend/src/container/GridGraphLayout/Graph/FullView/GraphManager.styles.scss

```diff
@@ -0,0 +1,21 @@
+.filter-save-container {
```

Comment:
I am not sure but these classes would be available in global can we wrap them under one class name so it does not collide with global CSS or mess with same class name

Reply:
Sure.